u64_stats_fetch_*_irq() variants were recently removed and are not
available anymore in fedora's 6.3 builds. See:
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=068c38ad88ccb09e5e966d4db5cedab0e02b3b95
---
 src/ice_main.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/ice_main.c b/src/ice_main.c
index 7bdd417..88b11fc 100644
--- a/src/ice_main.c
+++ b/src/ice_main.c
@@ -8555,10 +8555,10 @@ ice_fetch_u64_stats_per_ring(struct ice_ring_stats *ring_stat, u64 *pkts,
 	if (!ring_stat)
 		return;
 	do {
-		start = u64_stats_fetch_begin_irq(&ring_stat->syncp);
+		start = u64_stats_fetch_begin(&ring_stat->syncp);
 		*pkts = ring_stat->stats.pkts;
 		*bytes = ring_stat->stats.bytes;
-	} while (u64_stats_fetch_retry_irq(&ring_stat->syncp, start));
+	} while (u64_stats_fetch_retry(&ring_stat->syncp, start));
 }
 
 /**
-- 

